File: PRACTICA PRIMER PARCIAL/Ejercicio21/Python/SumaCalculadora.py
def calcular_suma(n):
    suma = 0
    for i in range(1, n+1):
        termino = (2 * i ** 2) / 5
        suma += termino
    return suma

n = 10  
resultado = calcular_suma(n)
print("Resultado:", resultado)

File: PRACTICA PRIMER PARCIAL/Ejercicio21/Java/SumaCalculadora.java
public class SumaCalculadora {
    public static void main(String[] args) {
        int n = 10;
        double resultado = calcularSuma(n);
        System.out.println("Resultado: " + resultado);
    }

    public static double calcularSuma(int n) {
        double suma = 0;
        for (int i = 1; i <= n; i++) {
            double termino = (2 * Math.pow(i, 2)) / 5.0;
            suma += termino;
        }
        return suma;
    }
}